api/package.json

```diff
@@ -86,6 +86,6 @@
     "winston-daily-rotate-file": "^4.5.0"
   },
   "engines": {
-    "node": ">14.19.0 <15.0.0"
+    "node": "14.21.1"
```

Comment:
I think the previous spec in here was invalid (needs a comma?).  Went ahead and specified a single version that is supported for now.